src/components/Success.vue

```diff
@@ -1,29 +1,19 @@
 <template>
     <div class="success center">
         <div class="icon-checkmark-circle"></div>
-        <h1>Key File export<br>was successfull!</h1>
+        <h1>{{ successText }}</h1>
         <div style="flex-grow: 1;"></div>
-        <button @click="close">Back to {{ request.appName }}</button>
+        <button @click="close">Back to {{ appName }}</button>
     </div>
 </template>
 
 <script lang="ts">
-import {Component, Emit, Vue} from 'vue-property-decorator';
-import {ParsedExportFileRequest} from '../lib/RequestTypes';
-import {State} from 'vuex-class';
-import {ResponseStatus, State as RpcState} from '@nimiq/rpc';
-import {ExportFileResult} from '@nimiq/keyguard-client';
-import { Static } from '@/lib/StaticStore';
+import {Component, Vue, Prop} from 'vue-property-decorator';
 @Component({components: {}})
-export default class ExportFileSuccess extends Vue {
-    @Static private request!: ParsedExportFileRequest;
-    @Static private rpcState!: RpcState;
-    @State private keyguardResult!: ExportFileResult;
-
-    @Emit()
-    private close() {
-        this.rpcState.reply(ResponseStatus.OK, this.keyguardResult);
-    }
+export default class Success extends Vue {
+    @Prop(String) private successText?: string;
+    @Prop(String) private appName?: string;
+    @Prop(Function) private close?: () => void;
```

Comment:
Instead of a prop, `close` should be an event:

```diff
-import {Component, Vue, Prop} from 'vue-property-decorator';
+import {Component, Emit, Vue, Prop} from 'vue-property-decorator';
```

```typescript
@Emit()
private close() {}
```

Reply:
Yes. Makes much more sense to have it as Event.